src/SUMA/SUMA_3dHist.c

```diff
@@ -621,7 +621,7 @@ int main(int argc, char **argv)
          if (!strcmp(Opt->feats->str[ii], "upvol")) {
             float vv;
             if (Opt->f1 < 0) {
-               SUMA_S_Err( "Bad news seeding voxvol");
+               SUMA_S_Err( "Bad news setting voxvol");
```

Comment:
@afni-rickr this one and below -- I took my form `setting`since I have not spotted anything about `seeding` in the code, but I could have been wrong -- please let me know if needs to be fixed back.